Contents/mods/CommunityAPI/media/lua/client/ModSettingAPI/ModSettingAPIClient.lua

```diff
@@ -0,0 +1,107 @@
+require("ModSettingAPI/ModSetting")
+require("ModSettingAPI/ModSettingPanel")
+local jsonUtils = require("CommunityAPI/JsonUtils")
+
+---@class ModSettingAPI
+ModSettingAPI = {}
```

Comment:
The main API class must be local and returned at the end of the file to be later used with `CommunityAPI.Client.ModSetting.MyFunction()`